install/systemd.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Run Airsonic with Systemd
+permalink: /docs/install/systemd/
```

Comment:
I prefer to add this to the standalone page or add it as a subcategory of standalone.

We currently only use systemd for the standalone installation.